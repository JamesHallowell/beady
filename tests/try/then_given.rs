use beady::scenario;

#[scenario]
#[test]
fn given_then_is_invalid() {
    'given_something: {
        'when_something: {
            'then_something: {
                'given_something: {
                    assert!(true);
                }
            }
        }
    }
}

fn main() {}
