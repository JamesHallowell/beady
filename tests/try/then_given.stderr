error: Only "and_then" is allowed inside "then" section
 --> tests/try/then_given.rs:9:17
  |
9 |                 'given_something: {
  |                 ^^^^^^^^^^^^^^^^
